src/atoms/feedCardItem/types.ts

```diff
@@ -156,12 +159,18 @@ export interface LpPool {
   token0_amount: number;
   token0_amount_usd: number;
   token0_price_usd: number;
-  token1_address: string;
-  token1_name: string;
-  token1_symbol: string;
-  token1_amount: number;
-  token1_amount_usd: number;
-  token1_price_usd: number;
+  token1_address?: string;
```

Comment:
@shan8851  everything is optional ?

Reply:
Yeah so for lp transaction types we have cases where we have between 1 and 3 tokens